doc/nrf/protocols/wifi/debugging.rst

```diff
@@ -28,22 +28,26 @@ With west
 
 .. code-block:: console
 
-    west build -p -b nrf7002dk/nrf5340/cpuapp samples/wifi/shell -- -Dnrf_wifi_shell_SNIPPET="nrf70-debug"
+    west build -p -b nrf7002dk/nrf5340/cpuapp samples/wifi/shell -- -Dnrf_wifi_shell_SNIPPET="nrf70-driver-verbose-debug;wpa-supplicant-debug"
```

Comment:
We should have sections to explain and demonstrate various debug snippets, combining them in to a single one won't work most of the times due to memory and defeats the purpose of splitting them up.